src/main/ipc.ts

```diff
@@ -81,6 +82,10 @@ const initialiseIpcHandlers: (ipcContext: IpcContext) => void = (
     writeRecentProjects(recentProjects)
   );
   // END GENERATED CODE PART 2
+  ipcMain.handle('user-os', async () => handleOsQuery());
```

Comment:
Is this line a duplicate from a merge conflict?

Reply:
yep. removed